actions/manifest-build/action.yml

```diff
@@ -0,0 +1,75 @@
+name: manifest-build
+description: "Builds all manifest models and uploads them to a cache"
+
+inputs:
+  # general inputs
+  cache-name:
+    description: "Cachix private cache name"
+    required: false
+    default: "manifest"
+  # grafana cloud inputs
+  metrics-job-name:
+    description: "grafana metrics job name"
+    required: false
+    default: manifest-build
+  gc-host:
+    description: "grafana cloud hostname"
+    required: false
+  gc-basic-auth:
+    description: "grafana cloud basic auth"
+    required: false
+secrets:
+  github-token:
+    description: "GitHub auth token for private repos (if needed)"
+    required: false
+    default: ""
+  cache-token:
+      description: "Cachix auth token for private cache"
+      required: false
+      default: ""
```

Comment:
Should we add an S3 option as well or is it too early?

Reply:
I think we should hold off and add that once we can test it e2e